packages/loot-core/src/server/accounts/parse-file.ts

```diff
@@ -137,7 +137,7 @@ async function parseOFX(
     errors,
     transactions: data.transactions.map(trans => {
       return {
-        amount: Number(trans.amount),
+        amount: trans.amount,
```

Comment:
Dont use the built in parser, which cant handle other format types other than decimal comma.